src/pages/admin/theme.js

```diff
@@ -0,0 +1,368 @@
+import { Button, Box } from '@mui/material';
+import { useKeycloak } from '@react-keycloak/ssr';
+import axios from 'axios';
+import log from 'loglevel';
+import React from 'react';
+import { buildTheme } from '../../context/themeContext';
+import useLocalStorage from '../../hooks/useLocalStorage';
+
+function ThemeConfig() {
+  const { keycloak } = useKeycloak();
+  const [RPT, setRPT] = React.useState(null);
+  // const [RPTToken, setRPTToken] = React.useState(null);
+  const [auz, setAuz] = React.useState(null);
+  const [theme, setTheme] = React.useState(
+    JSON.stringify(buildTheme('light'), null, 2),
+  );
+  const [key, setKey] = React.useState(1);
+  const [user, setUser] = React.useState(null);
+  // log.warn('theme to config:', theme);
+
+  const [themeObject, setThemeObject] = useLocalStorage(
+    'themeObject',
+    undefined,
+  );
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    setTheme(value);
+  }
+
+  function handlePreview() {
+    const themeObject2 = JSON.parse(theme);
+    setThemeObject(themeObject2);
+    setKey(key + 1);
+  }
+
+  function handleSave() {
+    // post theme to server
+    const url = `${process.env.NEXT_PUBLIC_CONFIG_API}/organizations/${user.organization_id}/theme`;
+
+    axios({
+      method: 'POST',
+      url,
+      data: {
+        theme,
+      },
+      headers: {
+        // Authorization: `Bearer ${keycloak.token}`,
+        Authorization: `Bearer ${auz.rpt}`,
+      },
+    })
+      .then((response) => {
+        log.warn('response:', response);
+        alert('Theme saved!');
+      })
+      .catch((error) => {
+        log.warn('error:', error);
+        alert('Theme save failed!');
+      });
+  }
```

Comment:
Please ignore all things about auth, keycloak, the real thing is here: save the theme object to server